providers/summary/clearlydefined.js

```diff
@@ -215,6 +215,32 @@ class ClearlyDescribedSummarizer {
     mergeDefinitions(result, newDefinition, get(result, 'licensed.declared') === 'OTHER')
   }
 
+  parseLicenseExpression(manifest, packageType) {
+    const combineLicenses = (exp, license) => {
+      if (exp) {
+        return exp + ' ' + (packageType==='npm'?'AND':'OR') + ' ' +
```

Comment:
Could you please enable Prettier & ESLint plugins in VSCode for this repo? It should format everything including spacing between ===, ?, : etc. Just want to make sure everything is consistent.
https://docs.clearlydefined.io/contributing-code

Reply:
I think I have fixed the offending lines.. Could you please verify?